uaclient/cli.py

```diff
@@ -194,9 +207,7 @@ def action_enable(args, cfg):
     logging.debug(ua_status.MESSAGE_REFRESH_ENABLE)
     if not contract.request_updated_contract(cfg):
         logging.debug(ua_status.MESSAGE_REFRESH_FAILURE)
-    ent_cls = entitlements.ENTITLEMENT_CLASS_BY_NAME[args.name]
-    entitlement = ent_cls(cfg)
-    return 0 if entitlement.enable() else 1
+    return 0 if _perform_enable(args.name, cfg) else 1
```

Comment:
Why do we want to switch to this call instead of calling the entitlement instance that we have locally in this function?

Reply:
The code here was refactored in to `_perform_enable` so it could be used from both the new code in `action_attach` and in `action_enable` (which also does refreshing, and could feasibly also introduce extra output in the future that `action_attach` shouldn't output).

Reply:
Makes sense. Thanks

Reply:
Just to round this out with our IRC conversation: I agree that `.enable(silent=True)` would be a reasonable path forward if we do end up wanting to suppress that output. During attach, however, we _do_ want the output that `.enable()` produces.

(This separate function would still be necessary so that we don't perform a spurious refresh (or two) during attach, and to avoid wasting effort by running through `assert_attached_root` multiple times.)